gradle/libs.versions.toml

```diff
@@ -120,6 +120,7 @@ glide-compiler = { module = "com.github.bumptech.glide:ksp", version.ref = "glid
 glide-core = { module = "com.github.bumptech.glide:glide", version.ref = "glide" }
 glide-okhttp3-integration = { module = "com.github.bumptech.glide:okhttp3-integration", version.ref = "glide" }
 gson = { module = "com.google.code.gson:gson", version.ref = "gson" }
+kapt = { module = "androidx.room:room-compiler", version.ref = "androidx-room" }
```

Comment:
Not sure why this is needed _now_ (starting today)?

If I don't have this there will be a build error that the (automatically generated) AppDatabase_Impl does not implement all needed dao factory methods.

Reply:
This should not be necessary as we migrated Room to ksp. If you get an error, try a clean build.

Reply:
Removed.